docs/MigrationGuide.md

```diff
@@ -22,6 +22,9 @@ To see the biggest differences please consult the [changelog](Changelog.md).
   now returns `ValidatedNel` with scenario compilation errors instead of throwing exception in that case 
 * [#3701](https://github.com/TouK/nussknacker/pull/3701) Change: SPeL TypedMap Get Accessor:
   * Right now access to not existing field on TypedMap won't throw exception, just will return `null`
+* [#3675](https://github.com/TouK/nussknacker/pull/3675) Improvements: Normalize kafka components params name
```

Comment:
please be precise which source's and sinks does this PR involve, if sources from dev-model IMO this note is redundant, or at least add info that's only dev-model sources and sinks, otherwise we need a migration